.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build
+
+on: push
```

Comment:
Do you want this to be _every_ push? I think `pull_request` to master is better. And then you can make `master` a protected branch requiring a PR and CI to succeed